.env.sample

```diff
@@ -0,0 +1,14 @@
+GOERLI_MNEMONIC="your goerli mnemonic"
+GOERLI_PROVIDER_URL="provider url"
+GOERLI_API_KEY="provider api key"
+MUMBAI_MNEMONIC="your mumbai mnemonic"
+MUMBAI_PROVIDER_URL="provider url"
+MUMBAI_API_KEY="provider api key"
+# configure token mint/approve amount and fee handlers
+TOKEN_AMOUNT="93274"
+FEE_ORACLE_ADDRESS="0x534aB5328d0447d6B13c5837D4F3C95ee0F108A2"
+# fee handler with oracle
+FHWO_GAS_USED="4444"
+FHWO_FEE_PERCENTAGE=666
+# basic fee handler, fee is in ETH
+BFH_FEE=0.3
```

Comment:
Can we please set this to something more reasonable as 0.001?  We don't have that much test tokens :smile: 
Also, check other values so they make sense. (Token amount 93274?)

Reply:
Or are these values not used at all and are just an example. Forgot how env sample works.